src/audio-worklet/free-queue/examples/simple-passthrough/main.js

```diff
@@ -0,0 +1,79 @@
+import FreeQueue from "../../src/free-queue.js"
+import { QUEUE_SIZE } from "./constants.js";
+
+const toogleButton = document.getElementById("toogle");
+toogleButton.disabled = false;
+
+// Create 2 FreeQueue instances with 4096 buffer length and 1 channel.
+const inputQueue = new FreeQueue(QUEUE_SIZE, 1);
+const outputQueue = new FreeQueue(QUEUE_SIZE, 1);
+// Create an atomic state for synchronization between worker and AudioWorklet.
+const atomicState = new Int32Array(
+    new SharedArrayBuffer(1 * Int32Array.BYTES_PER_ELEMENT)
+);
+
+let audioContext;
```

Comment:
It's also being used in toogle button click event handler.

Like if we haven't created/initialized it, on first click audio context will be created.
and if it alread exists, then on subsequent clicks, it will toogle between state of playing and suspended

Reply:
Then now we have 2 AudioContexts: here and L23. Do we need both?

Reply:
I think for state management we do need it rn. Can't think of any other design pattern.
Though this more like a global state holding current audio context.